openmc/mgxs/mgxs.py

```diff
@@ -1998,7 +1998,7 @@ def tallies(self):
         """
 
         # Instantiate tallies if they do not exist
-        if super(NuScatterMatrixXS, self).tallies is None:
+        if super(ScatterMatrixXS, self).tallies is None:
```

Comment:
I guess I didn't catch this on the first time around -- why not just `if self._tallies is None`?